app/assets/json/translations/locale-en.json

```diff
@@ -789,6 +794,14 @@
         "WELCOME_FILIPINO": "Filipino",
         "WELCOME_SPANISH": "Español"
     },
+    "PREFERENCES": {
+        "ALICE_GRIFFITH": {
+            "TITLE": "Alice Griffith Housing Development Resident Preference",
+            "DESC": "For households in which at least one member was a resident of the Alice Griffith housing development. This includes baseline and current residents that lived in the targeted redevelopment site on or after the time of application for Choice Neighborhoods of October 26, 2010.",
```

Comment:
@creativecoder is there a source of truth for this description text that we can verify against? couldn't find this text in the InVision screens linked in the Pivotal story or in the Pivotal story itself.

Reply:
@callaghanc Good question, @chrisdolendo created this translation text when starting the story. I believe it's supposed to match the preference description in Salesforce, but we put it here so it can be translated: https://sfhousing--full.cs95.my.salesforce.com/a0m0P00000yuzO0

Reply:
ok thanks